.github/workflows/test.yaml

```diff
@@ -26,11 +26,9 @@ jobs:
       - name: Build
         run: |
           BUILD_VERSION="0.0.0-${{ github.ref_name }}-$(git rev-parse --short $GITHUB_SHA)" pnpm build
-
       - name: Lint
         run: |
           pnpm lint
-
       - name: Checking PR Number
```

Comment:
When we merge this PR, we still want the rest of the repo to use this old `test` action instead of the new one. More context here: https://github.com/FuelLabs/fuels-ts/pull/728#issuecomment-1446348803

That's why I have kept it intact still.